requirements.txt

```diff
@@ -12,3 +12,4 @@ pytz==2015.4
 Flask-WTF==0.12
 markdown==2.6.2
 Flask-Script==2.0.5
+workdays==1.4
```

Comment:
Ah nice, maybe one for another story in the future. I guess we can get a pretty reliable list from [here](https://www.gov.uk/bank-holidays).